src/elements/content-explorer/MoreOptions.js

```diff
@@ -1,13 +1,16 @@
 // @flow
 import React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
```

Comment:
Should we replace all the `FormattedMessage`s with `intl.formatMessage()`? Like in the other accessibility PRs

Reply:
I think that when they're in the JSX directly instead of as a prop, it's safer to keep them as `FormattedMessage`, because it creates a `<span />` wrapper. Removing it could potentially affect some styles.